<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DOCX to HTML Converter</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow {
      background-color: #f5f5f5;
    }
    QGroupBox {
      font-weight: bold;
      border: 1px solid #e5e7eb;
      border-radius: 6px;
      margin-top: 12px;
      padding-top: 10px;
      background-color: #ffffff;
    }
    QGroupBox::title {
      subcontrol-origin: margin;
      left: 15px;
      padding: 0 8px;
      color: #1f2937;
    }
    QPushButton {
      padding: 10px 20px;
      border-radius: 5px;
      font-size: 13px;
      border: none;
    }
    QPushButton#convertBtn {
      background-color: #2563eb;
      color: white;
      font-weight: bold;
      font-size: 14px;
      padding: 12px 30px;
    }
    QPushButton#convertBtn:hover {
      background-color: #1d4ed8;
    }
    QPushButton#convertBtn:disabled {
      background-color: #93c5fd;
    }
    QPushButton#cancelBtn {
      background-color: #dc2626;
      color: white;
      padding: 12px 30px;
    }
    QPushButton#cancelBtn:hover {
      background-color: #b91c1c;
    }
    QPushButton#cancelBtn:disabled {
      background-color: #fca5a5;
    }
    QPushButton#browseBtn {
      background-color: #e5e7eb;
      color: #1f2937;
    }
    QPushButton#browseBtn:hover {
      background-color: #d1d5db;
    }
    QLineEdit {
      padding: 8px 12px;
      border: 1px solid #e5e7eb;
      border-radius: 5px;
      background-color: white;
    }
    QLineEdit:focus {
      border-color: #2563eb;
    }
    QTextEdit#logText {
      background-color: #1f2937;
      color: #e5e7eb;
      font-family: Consolas, monospace;
      font-size: 12px;
      border: none;
      border-radius: 5px;
      padding: 10px;
    }
    QProgressBar {
      border: none;
      border-radius: 5px;
      background-color: #e5e7eb;
      text-align: center;
    }
    QProgressBar::chunk {
      background-color: #2563eb;
      border-radius: 5px;
    }
    QProgressBar#totalProgress::chunk {
      background-color: #16a34a;
    }
    QRadioButton, QCheckBox {
      font-size: 13px;
      color: #1f2937;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <!-- Header -->
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>DOCX → HTML Converter</string>
      </property>
      <property name="styleSheet">
       <string>font-size: 22px; font-weight: bold; color: #2563eb;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="subtitleLabel">
      <property name="text">
       <string>Word 문서를 웹북용 HTML로 변환합니다</string>
      </property>
      <property name="styleSheet">
       <string>font-size: 12px; color: #6b7280; margin-bottom: 10px;</string>
      </property>
     </widget>
    </item>
    <!-- Mode Selection -->
    <item>
     <widget class="QGroupBox" name="modeGroup">
      <property name="title">
       <string>변환 모드</string>
      </property>
      <layout class="QHBoxLayout" name="modeLayout">
       <item>
        <widget class="QRadioButton" name="singleModeRadio">
         <property name="text">
          <string>단일 파일 변환</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="batchModeRadio">
         <property name="text">
          <string>배치 변환 (폴더 전체)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="modeSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <!-- Single File Frame -->
    <item>
     <widget class="QWidget" name="singleFrame">
      <layout class="QVBoxLayout" name="singleLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="inputFileGroup">
         <property name="title">
          <string>입력 파일</string>
         </property>
         <layout class="QHBoxLayout" name="inputFileLayout">
          <item>
           <widget class="QLineEdit" name="inputFileEdit">
            <property name="placeholderText">
             <string>Word 파일(.docx)을 선택하세요</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="inputFileBrowseBtn">
            <property name="objectName">
             <string>browseBtn</string>
            </property>
            <property name="text">
             <string>파일 선택</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="outputFileGroup">
         <property name="title">
          <string>출력 위치</string>
         </property>
         <layout class="QHBoxLayout" name="outputFileLayout">
          <item>
           <widget class="QLineEdit" name="outputFileEdit">
            <property name="placeholderText">
             <string>저장할 위치를 선택하세요</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="outputFileBrowseBtn">
            <property name="objectName">
             <string>browseBtn</string>
            </property>
            <property name="text">
             <string>위치 선택</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <!-- Batch Frame (initially hidden) -->
    <item>
     <widget class="QWidget" name="batchFrame">
      <property name="visible">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="batchLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="inputFolderGroup">
         <property name="title">
          <string>입력 폴더</string>
         </property>
         <layout class="QHBoxLayout" name="inputFolderLayout">
          <item>
           <widget class="QLineEdit" name="inputFolderEdit">
            <property name="placeholderText">
             <string>Word 파일이 있는 폴더를 선택하세요</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="inputFolderBrowseBtn">
            <property name="objectName">
             <string>browseBtn</string>
            </property>
            <property name="text">
             <string>폴더 선택</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="outputFolderGroup">
         <property name="title">
          <string>출력 폴더</string>
         </property>
         <layout class="QHBoxLayout" name="outputFolderLayout">
          <item>
           <widget class="QLineEdit" name="outputFolderEdit">
            <property name="placeholderText">
             <string>변환된 파일을 저장할 폴더를 선택하세요</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="outputFolderBrowseBtn">
            <property name="objectName">
             <string>browseBtn</string>
            </property>
            <property name="text">
             <string>폴더 선택</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="batchOptionsGroup">
         <property name="title">
          <string>배치 옵션</string>
         </property>
         <layout class="QHBoxLayout" name="batchOptionsLayout">
          <item>
           <widget class="QCheckBox" name="includeSubfoldersCheck">
            <property name="text">
             <string>하위 폴더 포함</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="batchOptionsSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="previewFilesBtn">
            <property name="objectName">
             <string>browseBtn</string>
            </property>
            <property name="text">
             <string>파일 목록 미리보기</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <!-- Options -->
    <item>
     <widget class="QGroupBox" name="optionsGroup">
      <property name="title">
       <string>변환 옵션</string>
      </property>
      <layout class="QHBoxLayout" name="optionsLayout">
       <item>
        <widget class="QCheckBox" name="extractImagesCheck">
         <property name="text">
          <string>이미지 추출</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="removeEmptyCheck">
         <property name="text">
          <string>빈 문단 제거</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="optionsSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <!-- Buttons -->
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <spacer name="buttonSpacerLeft">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="convertBtn">
        <property name="text">
         <string>▶  변환 실행</string>
        </property>
        <property name="minimumWidth">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelBtn">
        <property name="text">
         <string>■  취소</string>
        </property>
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumWidth">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="buttonSpacerRight">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <!-- Progress -->
    <item>
     <widget class="QGroupBox" name="progressGroup">
      <property name="title">
       <string>진행 상황</string>
      </property>
      <layout class="QVBoxLayout" name="progressLayout">
       <item>
        <widget class="QLabel" name="currentFileLabel">
         <property name="text">
          <string>대기 중...</string>
         </property>
         <property name="styleSheet">
          <string>color: #6b7280;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="currentProgress">
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalProgressLabel">
         <property name="text">
          <string>전체: 0/0</string>
         </property>
         <property name="styleSheet">
          <string>color: #6b7280; margin-top: 5px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="totalProgress">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <!-- Log -->
    <item>
     <widget class="QGroupBox" name="logGroup">
      <property name="title">
       <string>로그</string>
      </property>
      <layout class="QVBoxLayout" name="logLayout">
       <item>
        <widget class="QTextEdit" name="logText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="minimumHeight">
          <number>120</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <!-- Export Button -->
    <item>
     <layout class="QHBoxLayout" name="exportLayout">
      <item>
       <widget class="QPushButton" name="exportCsvBtn">
        <property name="objectName">
         <string>browseBtn</string>
        </property>
        <property name="text">
         <string>결과 CSV 내보내기</string>
        </property>
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="exportSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
